commit 5e10bfb78bffcf155c18133a74f094411ea6ee64
Author: Harald Sitter <sitter@kde.org>
Date:   Thu Jun 22 11:43:25 2023 +0200

    be explicit about default values
    
    0=Qt::Widget
    
    this also fixes build problems because we are trying to coerce an int to
    enum implicitly

diff --git a/demos/simplecapture/capture.h b/demos/simplecapture/capture.h
index 4e3bf56d..6da924f5 100644
--- a/demos/simplecapture/capture.h
+++ b/demos/simplecapture/capture.h
@@ -38,7 +38,7 @@ namespace Phonon {
  *
  * Can capture video using a media object.
  *
- * Using just a single media object, you can't capture audio and video 
+ * Using just a single media object, you can't capture audio and video
  * at the same time, if the desired device knows only audio or only video.
  *
  * @note Phonon capture is still not stabilised, don't be surprised if audio
@@ -60,7 +60,7 @@ public:
      *
      * @see setupCaptureSource
      */
-    CaptureWidget(QWidget *parent = NULL, Qt::WindowFlags f = 0);
+    CaptureWidget(QWidget *parent = NULL, Qt::WindowFlags f = Qt::Widget);
 
 private slots:
     /**
diff --git a/demos/simpleplayer/player.h b/demos/simpleplayer/player.h
index 334a8aa9..f8460701 100644
--- a/demos/simpleplayer/player.h
+++ b/demos/simpleplayer/player.h
@@ -38,7 +38,7 @@ class Player : public QWidget {
 Q_OBJECT
 
 public:
-    Player(QWidget *parent = 0, Qt::WindowFlags flag = 0);
+    Player(QWidget *parent = 0, Qt::WindowFlags flag = Qt::Widget);
 
 public slots:
 
